action.yml

```diff
@@ -4,12 +4,9 @@
 name: setup-enos
 description: Setup GitHub CLI, enos, on GitHub Actions runners
 inputs:
-  github-token:
```

Comment:
I don't want this change to break people. It would be nice if you could silently start using the PAT authenticated version. I'll test it to make sure we don't break people. If we do I'll make this an optional input with a message about it being nothing.

Reply:
I ended up swapping out the authentication strategy to unauthenticated. It means that old users of the action will see a warning when this version becomes the latest, but that seems fine.

![image](https://github.com/hashicorp/action-setup-enos/assets/65058/3df6b695-36aa-4dbe-9c37-b31537cedf12)

Reply:
Also tested it here and it works fine with and without any token set https://github.com/hashicorp-forge/terraform-provider-enos/actions/runs/8759769893/job/24043866943?pr=13#step:3:1